apps/api/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Use root/example as user/password credentials
+version: "3.1"
+
+services:
+    mongo:
+        image: mongo
+        restart: always
+        ports:
+            - "27017:27017"
+        environment:
+            MONGO_INITDB_ROOT_USERNAME: root
+            MONGO_INITDB_ROOT_PASSWORD: example
```

Comment:
Chore: could we look into using a named volume for persistent database storage?
Something like
```yaml
volumes:
  - type: volume
    source: mongodb_data_volume
    target: /data/db
```